src/utils.rs

```diff
@@ -4,19 +4,6 @@
 
 use ffi;
 use translate::*;
-use UserDirectory;
-
-pub fn get_application_name() -> Option<String> {
```

Comment:
I think we should do as follow: all standalone functions (so not methods of enums/structs) should be usable directly. As such, we should directly reexport `auto/functions`. (To be sure what I said was clear :p :

```rust
pub use functions::*;
```

So that we can call the functions like this:

```rust
gtk::some_func();
```
)